spring-integration-ip/src/main/java/org/springframework/integration/ip/tcp/connection/DefaultTcpSSLContextSupport.java

```diff
@@ -69,9 +77,28 @@ public DefaultTcpSSLContextSupport(String keyStore, String trustStore,
 		this.trustStorePassword = trustStorePassword.toCharArray();
 	}
 
+	/**
+	 * Set the key store type. Default JKS.
+	 * @param keyStoreType the type.
+	 * @since 5.0.8
+	 */
+	public void setKeyStoreType(String keyStoreType) {
+		this.keyStoreType = keyStoreType;
```

Comment:
I don't think that this can be as null or even empty: so, I will add `Assert.hasText(keyStoreType, "'keyStoreType' cannot be empty");`, as well as for the `setTrustStoreType()`

Reply:
Thanks; yes; I meant to add those asserts, but was distracted.